packages/react-components/react-drawer/src/components/DrawerInline/DrawerInline.types.ts

```diff
@@ -0,0 +1,45 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { DrawerBaseTypes } from '../../util/DrawerBase.types';
+
+export type DrawerInlineSlots = {
+  root: Slot<'div'>;
+};
+
+export type DrawerOpenChangeData = {
+  open: boolean;
+};
+
+/**
+ * DrawerInline Props
+ */
+export type DrawerInlineProps = ComponentProps<DrawerInlineSlots> &
+  DrawerBaseTypes & {
+    /**
+     * Controls the open state of the Drawer
+     *
+     * @default false
+     */
+    open?: boolean;
+
+    /**
+     * Default value for the uncontrolled open state of the Drawer.
+     *
+     * @default false
+     */
+    defaultOpen?: boolean;
+
+    /**
+     * Whether the drawer has a separator line.
+     * This prop only works when `type` is `inline`.
```

Comment:
Do you need here to add line about `inline` type? Cause it's already DrawerInline.types

Reply:
That's true. This was copied over from the base Drawer. I removed the line.